Simulation.jl

```diff
@@ -1,4 +1,10 @@
-function simulate_deposition(compositions_T, fcoeff, Ea_values, T, num_steps, dt)
+function meshgrid(x, y)
```

Comment:
I don't have any issue with keeping this for convenience, but FYI, see this discussion about this type of functionality: https://discourse.julialang.org/t/meshgrid-function-in-julia/48679/15